CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# [v0.14.0](https://github.com/upb-uc4/ui-web/compare/v0.13.1...v0.14.0) (Draft)
```

Comment:
```suggestion
# [v0.13.2 WIP](https://github.com/upb-uc4/ui-web/compare/v0.13.1...v0.14.0)
```
We tag our "next release" as the next subversion of the last sprint increment. E.g. we are in sprint 14 and hat a version v0.13.1, hence the nextr version will be v0.13.2. We will just bump the version on v0.14.0 on the release branch for the sprint increment.